File: PROYECTO/PSport-APP/PSport/Model/DTOs/PartidoDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PSport.Model.DTOs
{
    public class PartidoDTO
    {
        public Guid? id { get; set; }
        public string fecha { get; set; }
        public string jugadoresPartidos { get; set; }
        public string equiposPartidos { get; set; }
        public string torneoID { get; set; }
        public string nombreTorneo { get; set; }
        public string resultado { get; set; }
        public string ronda { get; set; }
    }
}

File: PROYECTO/PP-Sport-API-/PPSportAPI/src/main/kotlin/es/sergigavi/PPSportAPI/MODEL/DTO/PartidoDTO.kt
package es.sergigavi.PPSportAPI.MODEL.DTO

import java.time.LocalDateTime
import java.util.*

data class PartidoDTO(
    val id: UUID,
    val fecha: LocalDateTime?,
    val jugadoresPartidos :MutableSet<JugadorPartidoPDTO>,
    val equiposPartidos:MutableSet<EquipoPartidoDTO>,
    val torneoID: UUID?,
    val nombreTorneo:String?,
    var resultado:String?,
    val ronda:Int
)